src/linear_samplers/linear_sampler_helpers/hadamard.jl

```diff
@@ -0,0 +1,46 @@
+"""
+fwht!(x::AbstractVector; signs=ones(Bool, size(x)), scaling = 1)  
+
+Performs an in-place Fast Walsh Hadamard Transform (FWHT). `signs` allows the user to input a boolean vector that can 
+flip the signs in the transform. `scaling` allows the user to scale the result of the transform. 
+
+"""
+function fwht!(x::AbstractVector; signs=ones(Bool, size(x)), scaling = 1)
+    ln = size(x,1)
```

Comment:
You use `size` differently for the same quantity. Not the biggest deal, but need to think about consistency for v0.2

Reply:
Good catch. I have made this consistent between the two by using `size(x,1)`.